spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/BindableProxyFactory.java

```diff
@@ -62,7 +66,7 @@
  * @see EnableBinding
  */
 public class BindableProxyFactory implements MethodInterceptor, FactoryBean<Object>, Bindable, BeanFactoryAware,
```

Comment:
The BindableProxyFactory is created for each bound interface. I don't think that the error binding should happen within this class.